pkg/controller/controller.go

```diff
@@ -238,14 +238,14 @@ func (c *Controller) processPvcItem(pvc *v1.PersistentVolumeClaim) error {
 	}
 
 	// all checks passed, let's create the importer pod!
-	pod, err := c.createImporterPod(ep, secretName, pvc)
+	pod, err := CreateImporterPod(c.clientset, c.importerImage, c.verbose, c.pullPolicy, ep, secretName, pvc)
```

Comment:
This now needs several controller fields passed. Does it _need_ to be a func?

Reply:
I think so, IMHO this should be a generic function that can be called from anywhere, I don't think we need the clientset though, thought I fixed that. Also I thought we discussed this one on the original PR and decided it was fine to pass in params? @copejon any thoughts on this one?

Reply:
I'm in favor of it being a func.  While it's accepting a lot of controller params, those are still pretty generic values

Reply:
I guess I could see c.CreateImporterPod keeping as a method but MakeImporterPodSpec should probably be exported func?  thoughts?

Reply:
For sure above @screeley44 .  I also agree w/ @copejon that these new parms are generic, so I can go either way.